Reportedly/Modules/Main/Home/HomeViewController.swift

```diff
@@ -250,19 +264,19 @@ extension HomeViewController {
         menuUserAvatarImageView.snp.makeConstraints {
             $0.height.width.equalTo(CGFloat.spacer7)
             $0.trailing.equalToSuperview().inset(CGFloat.spacer4)
-            $0.bottom.equalTo(menuHorizontalSeperatorView.snp.top).offset(-CGFloat.spacer5)
+            $0.bottom.equalTo(menuVerticalSeperatorView.snp.top).offset(-CGFloat.spacer5)
```

Comment:
Can we update like this to use positive value ?

`menuVerticalSeperatorView.snp.makeConstraints {
            $0.centerY.equalToSuperview().multipliedBy(0.4)
            $0.leading.equalToSuperview().offset(CGFloat.spacer4)
            $0.trailing.equalToSuperview().inset(CGFloat.spacer4)
            $0.height.equalTo(1)
            $0.top.equalTo(menuUserAvatarImageView.snp. bottom).offset(CGFloat.spacer5)
        }`

Reply:
yeah, after hearing your suggestion, I understood now, let me update